src/test/java/harness/utils/WorkspaceUtils.java

```diff
@@ -0,0 +1,48 @@
+package harness.utils;
+
+import bio.terra.cli.serialization.userfacing.UFWorkspace;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import harness.CRLJanitor;
+import harness.TestCommand;
+import harness.TestUser;
+
+/** Utilities for working with workspaces in CLI tests. */
+public class WorkspaceUtils {
+
+  /**
+   * Create a new workspace and register it with Janitor if this test is running in an environment
+   * where Janitor is enabled.
+   *
+   * @param workspaceCreator The user who owns the workspace and should be used to clean it up.
+   */
+  public static UFWorkspace createWorkspaceWithCleanup(TestUser workspaceCreator)
```

Comment:
Drive-by review. I'm also creating a WorkspaceUtils in my branch. :)

Can this be `createWorkspace` instead of `createWorkspaceWithCleanup`?

Reply:
honestly I'm a little surprised it didn't exist already. and sg, done